tests/test_browser.py

```diff
@@ -4954,3 +4954,18 @@ def test_wasm2js_fallback(self):
       open('test.html', 'w').write(html)
       os.remove('test.wasm') # Also delete the Wasm file to test that it is not attempted to be loaded.
       self.run_browser('test.html', 'hello!', '/report_result?0')
+
+  # Test that basic thread creation works in combination with MODULARIZE_INSTANCE=1 and EXPORT_NAME=MyModule
+  @no_fastcomp('more work would be needed for this to work in deprecated fastcomp')
```

Comment:
what is the difference with fastcomp that causes this to fail? Iiuc `MODULARIZE_INSTANCE` is an identical feature in both backends?

Reply:
Fastcomp emits a bunch of code in the HTML to deal with the asm.js memory init file, and that has hardcoded `Module` in there (instead of the custom export name). From the stack trace I think that's what goes wrong, but I didn't investigate.